README.md

```diff
@@ -26,118 +29,187 @@ GQL architecture is inspired by `React-Relay` and `Apollo-Client`.
 
     $ pip install gql
 
-Please note that the following documentation describes the current version which is currently only available as a pre-relase and needs to be installed with
+> **WARNING**: Please note that the following documentation describes the current version which is currently only available as a pre-release and needs to be installed with
 
     $ pip install --pre gql
 
 ## Usage
 
-The example below shows how you can execute queries against a local schema.
+### Basic usage
 
 ```python
-from gql import gql, Client
+from gql import gql, Client, AIOHTTPTransport
 
-from .someSchema import SampleSchema
+# Select your transport with a defined url endpoint
+transport = AIOHTTPTransport(url="https://countries.trevorblades.com/")
 
+# Create a GraphQL client using the defined transport
+client = Client(transport=transport, fetch_schema_from_transport=True)
```

Comment:
As basic usage, would be great to keep minimal the example code by excluding the transport in this case.

A new section below this one should explain the transport option with an example as currently does and mention the other available transports.

Reply:
I bet that more than 99% of users need the gql library to execute GraphQL queries from a server. Do you know one person which uses gql to execute a local schema ? That is the reason that I moved this functionality lower in the readme. I find it confusing if it is at first. IMHO the first example should be a very simple example of a query on an http transport like I did.

Reply:
You got a point.